### PR TITLE
feat: Wishlist screen development

### PR DESCRIPTION
**Pull request by dev-enforced**
In this pull request you will be able to view the screen that displays the list of products liked by the user of the ecommerce web app. 
Code changes involve:
1) Wishlist screen development
2) Shift of common classes from separate CSS files to base.css files
3) Providing links in all screens for accessibility (except authentication component)

Preview Link: https://deploy-preview-5--chillar-party.netlify.app/pages/wishlist.html

### CODE REVIEW

**Comment by rkapoor10** [comment]
make both the buttons of the same width for a better appearance.

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+                </div>
+              </div>
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <!-- Navigation ends -->
+      <section class="main-content">
+          <div class="main-title txt-sm gentle-flex flex-justify-center">
+              My Wishlist
```
Comment (rkapoor10) [inline pages/wishlist.html]: change the font color of "My WishList"

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                    <p class="card-pricing">
+                        <span class="card-price">₹600</span>
+                        <del class="card-original-price">₹1000</del>
+                        <span class="card-discount">40% off</span>
+                    </p>
+                </div>
+                <div class="card-actions">
+                    <button class="btn btn-warning btn-warning-hover">ADD TO CART</button>
+                    <button class="btn btn-outline">
+                        <a class="link-none" href=""> Remove From Wishlist </a>
```
Comment (rkapoor10) [inline pages/wishlist.html]: use the same case...ADD TO CART & Remove From Wishlist

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge...just go through the suggesstions

File: pages/wishlist.html (hunk 2 above)
Comment (dev-enforced) [reply pages/wishlist.html]: Same case meaning both shall be of same type?

File: pages/cart-management.html [hunk 3]
```diff
@@ -85,134 +85,134 @@
...
-                <div class="total-price gentle-flex-gap flex-space-between bd-bottom py-5">
-                    <h4>TOTAL AMOUNT</h4>
-                    <h4>1200</h4>
+              </div>
+              <div class="cart-actions flex-column gentle-flex-gap p-3">
+                <button class="btn btn-warning btn-warning-hover">
+                  <a class="link-none" href="">Remove From Cart</a>
+                </button>
+                <button class="btn btn-outline">
+                  <a class="link-none" href=""> Move To Wishlist </a>
```
Comment (harshitadaswani) [inline pages/cart-management.html]: add some value to `href` attribute

File: pages/cart-management.html [hunk 4]
```diff
@@ -85,134 +85,134 @@
...
-                        </a>
-                    </button>
-                </div>   
+              </div>
+              <div class="cart-actions flex-column gentle-flex-gap p-3">
+                <button class="btn btn-warning btn-warning-hover">
+                  <a class="link-none" href="">Remove From Cart</a>
+                </button>
+                <button class="btn btn-outline">
+                  <a class="link-none" href=""> Move To Wishlist </a>
```
Comment (harshitadaswani) [inline pages/cart-management.html]: same here

File: pages/wishlist.html [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+          <i class="fas fa-search search-icon"></i>
+          <input
+            type="search"
+            class="gentle-search-bar"
+            placeholder="Search-for-products"
+          />
+        </div>
+        <ul class="gentle-nav-links ecom-nav-links">
+          <li>
+            <a class="link-none" href=""> LOGIN </a>
```
Comment (harshitadaswani) [inline pages/wishlist.html]: same here

File: pages/wishlist.html [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+                    <p class="card-pricing">
+                        <span class="card-price">₹600</span>
+                        <del class="card-original-price">₹1000</del>
+                        <span class="card-discount">40% off</span>
+                    </p>
+                </div>
+                <div class="card-actions">
+                    <button class="btn btn-warning btn-warning-hover">ADD TO CART</button>
+                    <button class="btn btn-outline">
+                        <a class="link-none" href=""> Remove From Wishlist </a>
```
Comment (harshitadaswani) [inline pages/wishlist.html]: same here

File: pages/wishlist.html [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
```
Comment (harshitadaswani) [inline pages/wishlist.html]: same here

File: pages/wishlist.html [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
```
Comment (harshitadaswani) [inline pages/wishlist.html]: same here

File: pages/wishlist.html (hunk 2 above)
Comment (harshitadaswani) [reply pages/wishlist.html]: uppercase or lowercase

File: pages/wishlist.html [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+
+      </section>
+      <!-- Footer component begins -->
+      <footer class="footer-component">
+        <div class="footer-content-container p-8">
+          <div class="shop-details">
+            <div>
+              <img
+                src="assets/images/brand-logo.png"
+                alt=""
```
Comment (harshitadaswani) [inline pages/wishlist.html]: give alt

**Review by harshitadaswani — APPROVED** [review]
give values to attribute tags and then merge it

File: pages/wishlist.html (hunk 1 above)
Comment (dev-enforced) [reply pages/wishlist.html]: The color has been changed

File: pages/wishlist.html (hunk 2 above)
Comment (dev-enforced) [reply pages/wishlist.html]: Got it!!

File: pages/cart-management.html (hunk 3 above)
Comment (dev-enforced) [reply pages/cart-management.html]: Done

File: pages/cart-management.html (hunk 4 above)
Comment (dev-enforced) [reply pages/cart-management.html]: Done

File: pages/wishlist.html (hunk 5 above)
Comment (dev-enforced) [reply pages/wishlist.html]: Done

File: pages/wishlist.html (hunk 6 above)
Comment (dev-enforced) [reply pages/wishlist.html]: Done

File: pages/wishlist.html (hunk 7 above)
Comment (dev-enforced) [reply pages/wishlist.html]: Done

File: pages/wishlist.html (hunk 8 above)
Comment (dev-enforced) [reply pages/wishlist.html]: Done

File: pages/wishlist.html (hunk 9 above)
Comment (dev-enforced) [reply pages/wishlist.html]: Given

**Pull request merged**